### PR TITLE
fixing power limit display

### PR DESCRIPTION
**Pull request by discip**
@Ralim

Since the **first digit was cut off**, the displayed value was misleading  unless one was careful to notice that the scrollbar _**didn't blink**_ after **`95`** was reached.

Not sure if this solution is correct as the **`Pinecil V2`** side says max **88W**.
But after seeing the wiki I'm even more confused as there are several different statements about the maximum wattage allowed. Also, the current **`MAX_POWER_LIMIT is 220`**.

Do we need to lower the **`MAX_POWER_LIMIT`** or should this fix be implemented?

### CODE REVIEW

**Comment by River-Mochi** [comment]
Pinecil  says 88W  as safety for the manufacturer but can theoretically go to about 120w.
I have personally pushed it to  110W  in extreme testing.

TL;DR    manufacturer will only support it at 24V max,  which is the 88W as they are also worried about people using more than 24V  on the DC barrel jack;   unofficially,  it supports  PD3.1 usb-C  chargers to 28V 
[ohm calculator](https://ohmslawcalculator.com/ohms-law-calculator)  will show  ~126w theoretical max at 28v, 6.2ohm tip.  in reality we can't ever get 126w but we can get close.

they base the 88W   on    24V max   and a 6.5 ohm Tip resistance (again to be conservative from manufacturer so that people don't complain).

in reality most of the tips are  6.2 ohms.   24v , 6.2ohms  = theoretical max of ~93w
6.5 ohm , 24v = 88w (printed on the side to be conservative number from manufacturer. head of Pine64 personally told me this )

In reality, we have been testing Pinecil V2 for over 7 months now with special EPR PD3.1 chargers at 28V  which  gives Theoretical max of  126V.   my testing with PD3.1 28V  ,  I'm only getting about 110W  because V2 goes from  cold to  320C  so fast  2-3 seconds that it never  gets chance to do more than  the 110W I've pushed it to using special test methods.

**Review by Ralim — APPROVED** [review]

**Comment by discip** [comment]
@Ralim
Sorry to bring this up again, but I don't quite get why the **`MAX_POWER_LIMIT`** is **`220`**.
Shouldn't that be something like **`140`** according to https://github.com/Ralim/IronOS/pull/1641#issuecomment-1494917367?

**Comment by River-Mochi** [comment]
maybe it was typo?  220w is a lot.
